webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,999 @@
+        yaxis=dict(range=[-0.15, 0.15], title='', 
+                   showticklabels=False), 
+        xaxis=dict(range=[-0.23, x[-1]+0.23], 
+                   title='', 
+                   ticktext=parameters, 
+                   tickvals=[steps*i for i in range(num_arrows)] if num_arrows>1 else [1]),
+        hoverlabel=dict(
+            bgcolor="white", 
+        )
+    )
+    fig.add_annotation(
+        x=-0.23,
+        y=0,
+        text="Small <br>p-value",
+        showarrow=False
+    )
+    fig.add_annotation(
+        x=x[-1]+0.23,
+        y=0,
+        text="Great <br>p-value",
+        showarrow=False
+    )
+    fig["layout"].update(
+        theme_layout(
+            theme,
+            {
+                "barmode": "relative",
+                "height": 500,
+                "title": "Parameters impact (increase " #Usikker på tittel (særlig det i parentes)
+                         "or decrese) on response and "
+                         "their significance"
+            }
+        )
+    )
+    fig["layout"]["font"].update({"size": 12})
+
+    """Costumizing the hoverer"""
+    fig.update_traces(hovertemplate='%{x}') #x is ticktext
+
+    """Adding arrows to figure"""
+    for i, s in enumerate(sgn):
+        xx = x[i] if num_arrows>1 else x[1]
+        fig.add_shape(
+            type="path",
+            path=f" M {xx-0.025} 0 " \
+                 f" L {xx-0.025} {s*0.06} " \
+                 f" L {xx-0.07} {s*0.06} " \
+                 f" L {xx} {s*0.08} " \
+                 f" L {xx+0.07} {s*0.06} " \
+                 f" L {xx+0.025} {s*0.06} " \
+                 f" L {xx+0.025} 0 ",
+            line_color="#222A2A",
+            fillcolor="crimson" if p_values[i] < 0.05 else "#606060",
+            line_width=0.6  
+        )
+    
+    """Adding zero-line along y-axis"""
+    fig.add_shape(
+        type="line",
+        x0=-0.1,
+        y0=0,
+        x1=x[-1]+0.1,
+        y1=0,
+        line=dict(
+            color='#222A2A',
+            width=0.75,
+        ),
+    )
+    return fig
+
+def make_range_slider(domid, values, col_name):
+    try:
+        values.apply(pd.to_numeric, errors="raise")
+    except ValueError:
+        raise ValueError(
+            f"Cannot calculate filter range for {col_name}. "
+            "Ensure that it is a numerical column."
+        )
+    return dcc.RangeSlider(
+        id=domid,
+        min=values.min(),
+        max=values.max(),
+        step=calculate_slider_step(
+            min_value=values.min(),
+            max_value=values.max(),
+            steps=len(list(values.unique())) - 1,
+        ),
+        value=[values.min(), values.max()],
+        marks={
+            str(values.min()): {"label": f"{values.min():.2f}"},
+            str(values.max()): {"label": f"{values.max():.2f}"},
+        }
+    )
+
+def theme_layout(theme, specific_layout):
+    layout = {}
+    layout.update(theme["layout"])
+    layout.update(specific_layout)
+    return layout
```

Comment:
Instead of using this, you can now use the [create_themed_layout](https://github.com/equinor/webviz-config/blob/5f06334c1e98defebb9a5efcb00fde21867822d4/webviz_config/_theme_class.py#L78-L104) method directly on a webviz_theme (then your theme has to be stored as e.g. `self.theme = app.webviz_settings["theme"]` instead of as a `plotly_theme` on the top. This will do a "safe" merge of the specific layout and the existing theme, making sure that nothing is overwritten, which could happen with regular dictionary updates. You can e.g. see how it is used here: https://github.com/equinor/webviz-subsurface/blob/2636ff02c7924bdf50cb80538adb2187de0eee68/webviz_subsurface/plugins/_disk_usage.py#L107